src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] t would be better if the App is class-based. Kindly fix this minor issue.

```suggestion
class App extends Component {
  return (
```